coordinator/src/node.rs

```diff
@@ -194,6 +209,20 @@ impl Node {
             .propose_dlc_channel_collaborative_settlement(&channel_id, accept_settlement_amount)?;
 
         let mut connection = self.pool.get()?;
+        db::trades::insert(
```

Comment:
I know that at this point it is not guaranteed that the trade will be successful, but I guess this is good for now. We will see based on the `position_id` if a trade was successful or not.